assignments/this.js

```diff
@@ -13,14 +13,82 @@
 
 // code example for Window Binding
 
+function windowBind() {
+    console.log(this)
+}
+
+windowBind();
+
 // Principle 2
 
 // code example for Implicit Binding
 
+// const crossFitter = {
+//     'name': 'Jordan',
```

Comment:
I would limit the use of literal strings to object keys that aren't valid JS variables (e.g. keys with spaces or dashes -)... see https://github.com/airbnb/javascript#objects--quoted-props